src/cpu/flexcpu/SConscript

```diff
@@ -38,9 +38,14 @@ if 'SimpleDataflowCPU' in env['CPU_MODELS']:
     Source('inflight_inst.cc')
```

Comment:
I don't think so... I'd like to run some tests with it, but you can merge it now and I'll start the tests on flexcpu.